backend/lcfs/web/api/transaction/repo.py

```diff
@@ -264,7 +264,7 @@ async def calculate_available_balance_for_period(
             int: The available balance of compliance units for the specified organization and period. Returns 0 if no balance is calculated.
         """
         compliance_period_end = datetime.strptime(
-            f"{str(period + 1)}-12-31", "%Y-%m-%d"
+            f"{str(period + 1)}-03-31", "%Y-%m-%d"
```

Comment:
This seems like a temporary fix, why do we need to manually set it to march?

Reply:
Thanks for the review @dhaselhan, that's the requirement set to calculate the transactions for the compliance period as per comment here
https://github.com/bcgov/lcfs/issues/756#issuecomment-2349764800.

please let me know if this needs any modifications or corrections.

Reply:
ah okay, I see now. Its more that the year for the report isnt exactly a perfect year. Thanks for clarifying!